src/event_logger.rs

```diff
@@ -30,6 +30,7 @@ pub enum LoggerEventKind {
     AttinyCommError,
     Rp2040MissedAudioAlarm(u64),
     AudioRecordingFailed,
+    EmptyErase,
```

Comment:
Could we come up with a more descriptive name than `EmptyErase`?  Maybe something like `ErasePartialOrCorruptRecording` – Assuming that's what it's referring to?